api/app/helpers/spree/api/api_helpers.rb

```diff
@@ -116,7 +116,7 @@ def required_fields_for(model)
       @@address_attributes = [
         :id, :firstname, :lastname, :full_name, :address1, :address2, :city,
         :zipcode, :phone, :company, :alternative_phone, :country_id, :state_id,
-        :state_name, :state_text
+        :location_name, :state_name, :state_text
```

Comment:
@MatthewKennedy could we call this new field `label`? That's probably the most universal one

Reply:
I’ll make this change.